# PyCharm的远程解释器下有包，但是总是在本地ieda显示 该package不存在或者缺少某些module：

## 方案1、在确认连接到正确的解释器后清除IDE缓存

在 PyCharm 中，转到"File"（文件）菜单 -> "Invalidate Caches / Restart"（无效化缓存/重启）。
勾选 Clear file system cache and Local History
点击 INVALIDATE AND　RESTART　清除IDE的缓存，并重新启动IDE。

## 方案2、重新载入远程lib文件夹

在 PyCharm 中，转到 File -> Settings -> Project -> Python Interpreter；
选择远程解释器，点击最上排的文件夹树状标识，选中 site-packages 点击 - 标识； 
再通过 + 标识重新载入对应解释器的 site-packages 目录。
